code/game/jobs/jobs.dm

```diff
@@ -31,9 +31,10 @@ var/const/VIROLOGIST		=(1<<6)
 var/const/PSYCHIATRIST		=(1<<7)
 var/const/ROBOTICIST		=(1<<8)
 var/const/XENOBIOLOGIST		=(1<<9)
-var/const/MED_TECH			=(1<<10)
-var/const/INTERN_MED		=(1<<11)
-var/const/INTERN_SCI		=(1<<12)
+var/const/XENOBOTANIST		=(1<<10)
```

Comment:
Do not put this flag in between others: leave them with their previous values and put xenobotanist at the bottom with 1<<13 as a value, otherwise you are going to fuck up people's prefs.

Reply:
addressed.